search-app/src/cmr/search/data/keywords_to_elastic.clj

```diff
@@ -91,6 +91,19 @@
   (let [regex (process-keyword keyword)]
     {:regexp {field regex}}))
 
+(defn- keywords->regex-filter
+  "Create a filter for keyword searches that checks for a loose match on "
```

Comment:
Looks like you got interrupted while writing the docstring ... I'm guessing you'll compare/update the docstring here and the one for `keywords->name-filter`?